src/aac/hookspecs.py.a46ee02f3b9e4ce23bca44b210122dee.tmp

```diff
@@ -0,0 +1,23 @@
+"""Defines the AaC plugin interface via Pluggy Hookspecs."""
```

Comment:
Do we need to update the .gitignore to exclude .tmp files?

Reply:
I did, but this guy snuck in there. Thanks for catching it!